src/pathPlanning/LinePolygonsCrossings.cpp

```diff
@@ -144,29 +144,59 @@ void LinePolygonsCrossings::getBasicCombingPath(CombPath& combPath)
 
 void LinePolygonsCrossings::getBasicCombingPath(PolyCrossings& polyCrossings, CombPath& combPath) 
```

Comment:
I know the method name "getBasicCombingPath" is there for a long time, usually getters return value. What about rename to "generateBasicCombingPath"

Reply:
I don't mind. I can do that.